src/status_im/events.cljs

```diff
@@ -1044,3 +1046,9 @@
  [(re-frame/inject-cofx :random-id-generator)]
  (fn [cofx _]
    (contact/add-new-identity-to-contacts cofx)))
+
+(handlers/register-handler-fx
+ :desktop/handle-universal-link
+ (fn [cofx [_ url]]
+   (desktop.deep-links/on-url-opened)
```

Comment:
why is this needed here ?

Reply:
@yenda `on-url-opened' is needed to set up new callback for urlOpened event. We need to set up it each time after last callback was fired and destroyed.

Reply:
@dmitryn  Does it make more sense to listen to the custom native event all the time?
Implementation sample: NetInfo.js subscribes and listens to native custom events there https://github.com/status-im/react-native-desktop/blob/master/Libraries/Network/NetInfo.js#L19
The event is generated from the cpp code: https://github.com/status-im/react-native-desktop/blob/master/ReactQt/runtime/src/netinfo.cpp#L80

Reply:
@MaxRis yes, it makes sense. I will use that approach. Thanks.